integreat_cms/cms/templates/feedback/admin_feedback_list_archived.html

```diff
@@ -0,0 +1,166 @@
+                        </span>
+                        <span class="{% if not feedback.rating_sum_negative %}invisible{% endif %}">
+                            <i icon-name="thumbs-down" class="pl-1"></i> {{ feedback.rating_sum_negative }}
+                        </span>
+                    </td>
+                    <td class="pr-2">
+                        <a href="{% url 'dashboard' region_slug=feedback.region.slug %}"
+                           class="hover:underline">
+                            {{ feedback.region.get_administrative_division_display }} {{ feedback.region.name }}
+                        </a>
+                    </td>
+                    <td class="pr-2">
+                        {% if feedback.read_by.email %}<a href="mailto:{{ feedback.read_by.email }}" class="hover:underline">{% endif %}
+                        {{ feedback.read_by.full_user_name }}
+                        {% if feedback.read_by.email %}</a>{% endif %}
+                    </td>
+                    <td class="pr-2 text-right whitespace-nowrap">{{ feedback.created_date }}</td>
+                </tr>
+            {% empty %}
+                <tr>
+                    <td colspan="9" class="px-2 py-3 text-center">
+                        {% if filter_form.has_changed %}
+                            {% translate "No archived feedbacks found with these filters." %}
+                        {% else %}
+                            {% translate "No archived feedbacks available yet." %}
+                        {% endif %}
+                    </td>
+                </tr>
+            {% endfor %}
+        </table>
+        {% if admin_feedback %}
+            <div class="pt-2 px-2">
+                <div class="inline">
+                    <span class="text-gray-800 font-bold" data-list-selection-count>0</span> <span class="text-gray-600">{% translate "Feedback entries selected" %}</span>
+                </div>
+            </div>
+        {% endif %}
+        <div class="flex flex-col">
+            {% include "pagination.html" with chunk=admin_feedback %}
+            <div class="flex self-start gap-2 mt-2">
+                <select id="bulk-action">
+                    <option>
+                        {% translate "Select bulk action" %}
+                    </option>
+                    <option data-bulk-action="{% url 'restore_admin_feedback' %}">
+                        {% translate "Restore" %}
+                    </option>
+                    {% if perms.cms.delete_feedback %}
+                        <option data-bulk-action="{% url 'delete_admin_feedback' %}">
+                            {% translate "Delete" %}
+                        </option>
+                    {% endif %}
+                </select>
+                <button id="bulk-action-execute" class="btn" disabled>{% translate "Execute" %}</button>
+            </div>
```

Comment:
```suggestion
            {% if perms.cms.change_feedback or perms.cms.delete_feedback %}
                <div class="flex self-start gap-2 mt-2">
                    <select id="bulk-action">
                        <option>
                            {% translate "Select bulk action" %}
                        </option>
                        {% if perms.cms.change_feedback %}
                            <option data-bulk-action="{% url 'restore_admin_feedback' %}">
                                {% translate "Restore" %}
                            </option>
                        {% endif %}
                        {% if perms.cms.delete_feedback %}
                            <option data-bulk-action="{% url 'delete_admin_feedback' %}">
                                {% translate "Delete" %}
                            </option>
                        {% endif %}
                    </select>
                    <button id="bulk-action-execute" class="btn" disabled>{% translate "Execute" %}</button>
                </div>
            {% endif %}
```

Here it's similar but we can here additionally hide the entire bulk action for marketing team users because both the current options are not available for them.

Reply:
As I understand, if user has the delete_feedback permission, he also has the change_feedback permission.
So I only keep change_feedback condition.
Pls let me know if you actually think that both conditions are must.